addons/web/static/src/views/form/form_controller.js

```diff
@@ -136,6 +136,8 @@ export class FormController extends Component {
         preventEdit: { type: Boolean, optional: true },
         onDiscard: { type: Function, optional: true },
         onSave: { type: Function, optional: true },
+        // As a generic component, additional props are allowed to be propagated
+        "*": {},
```

Comment:
> I don't see "*": {} anywhere else in the codebase, either it's broken everywhere else... or this is wrong.
Here for example : 
![image](https://github.com/odoo/odoo/assets/22252071/c3bc54d9-ae7e-4ffe-987e-e2c7f4fc7e66)
And at other several places. And here the link to the documentation where I discovered this feature : https://github.com/odoo/owl/blob/68f491cd32339d3c6e3e35dd0902fe59ff966206/doc/reference/props.md?plain=1#L220

Reply:
Still don't see why that would be the right fix here :wink: I understand that `"*"` "allows any prop" but it feels like something that should not exist in normal cases. I actually don't understand why that DialogWrapper would accept any prop, it does not even use those :shrug: 
=>
> Why are we transferring those clonePage and deletePage as viewProps at all?

Reply:
> Why are we transferring those clonePage and deletePage as viewProps at all?

@qsm-odoo  before this FIX: d5db48b2c76b6f94480d3b35093ea08f2cbf01a0, the whole `"web.FormViewDialog"` template was patched to add extra buttons (to "duplicate" & "delete" the current page), which caused some style issues as far as I remember :thinking:.
Now, the buttons are added in the correct way: we inherit the `web.FormViewDialog.ToOne.buttons` button template instead of `web.FormViewDialog` and we pass it as a prop (`this.viewProps > buttonTemplate`) to the `formController` with the extra buttons added to it... Actually, this is why we need to pass the `clonePage` & `deletePage` handlers as props to the form controller, where the `buttonTemplate` template will be called.

After discussing this today with @SergeBayet, I was convinced that the current diff makes sense in this case, but after I checked the code, I think we can fix the component props validation in a more "Owl Views Structure Friendly" way:

**For stable [saas-17.1]**:

All we need here is to add an override of the controller that validates the props, something like:

```javascript
export class PagePropertiesFormController extends FormController {
    static props = {
        ...FormController.props,
        clonePage: { type: Function, optional: true },
        deletePage: { type: Function, optional: true },
    };
}

registry.category("views").add("page_properties_dialog_form", {
    ...formView,
    Controller: PagePropertiesFormController,
});

export class PagePropertiesDialog extends FormViewDialog {
    //...
    this.viewProps = {
        ...this.viewProps,
        type: "page_properties_dialog_form",
        /...
    };
    //...
};
```

**For master:**

You can completely remove the "props" in XML (`t-on-click="props.clonePage/deletePage"` > `t-on-click="clonePage/deletePage"`) and use them directly in the controller patch (to test) :+1::

```javascript
export class PagePropertiesFormController extends FormController {
    //...
    clonePage() {
        return this.props.clonePage();
    }
    //...
};
```

Reply:
> Actually, this is why we need to pass the clonePage & deletePage handlers as props to the form controller, where the buttonTemplate template will be called.

Ok I see, make sense

> I think we can fix the component props validation in a more "Owl Views Structure Friendly" way

Indeed if the clonePage & deletePage props are not used by the FormViewDialog / PagePropertiesDialog directly, that's what I would imagine we should do :+1: It seems better to use specialized components with proper props validation than saying that the generic component accept all props :+1:

Reply:
@qsm-odoo @xO-Tx This is indeed a better solution.

Reply:
@qsm-odoo I've force-pushed to meet the @xO-Tx solution.